generators/client/templates/angular/src/main/webapp/app/shared/date/format-medium-date.pipe.ts.ejs

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Can you add few unit tests to check that the date is correctly formatted ? That will help us if we switch lib like we did recently

Same goes for the other pipe 😄

Reply:
done